packages/core-kit/src/nodes/fetch.ts

```diff
@@ -118,12 +119,54 @@ export const fetchDescriber: NodeDescriberFunction = async () => {
           description: "The HTTP status code of the response",
           type: "number",
         },
+        statusText: {
+          title: "statusText",
+          description: "The status text of the response",
+          type: "string",
+        },
+        contentType: {
+          title: "contentType",
+          description: "The content type of the response",
+          type: "string",
+        },
+        responseHeaders: {
+          title: "responseHeaders",
+          description: "The headers of the response",
+          type: "object",
+          additionalProperties: {
+            type: "string",
+          },
+        },
+        responseSchema: {
+          title: "responseSchema",
+          description: "The JSON schema of the response",
+          type: "object",
+        },
       },
-      required: ["response", "status"],
+      required: ["response"],
     },
   };
 };
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function generateJsonSchemaFromObject(obj: any): Schema {
```

Comment:
use unknown here

Reply:
had that originally. but then TS is unhappy with recurse down https://github.com/ExaDev/breadboard/blob/fetch-node-outputs/packages/core-kit/src/nodes/fetch.ts#L161

Reply:
Maybe use `NodeValue` here?

Reply:
Tried that one as well
```bash
  No index signature with a parameter of type 'string' was found on type 'Capability | { [key: string]: NodeValue; }'.ts(7053)
```

Reply:
> had that originally. but then TS is unhappy with recurse down https://github.com/ExaDev/breadboard/blob/fetch-node-outputs/packages/core-kit/src/nodes/fetch.ts#L161

What error does it give? I would be surprised if it were unhappy with the conversion to "unknown"